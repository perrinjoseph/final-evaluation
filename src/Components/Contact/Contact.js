import React from "react";
import Button from "../Button/Button.js";

function Contact() {
  const handelClick = (e) => {
    e.preventDefault();
  };

  return (
    <div className="contact ">
      <h1 className="projects__heading__center">
        Contact
        <br></br>
        <small style={{ fontSize: "20px" }}>
          <i>"XXXX"</i>
        </small>
        <br></br>
        <small style={{ fontSize: "20px" }}>
          <i>"XXXX"</i>
        </small>
      </h1>
      <main className="contact__main">
        <div className="about__main ">
          <h3>Reaching out to me</h3>
          <p>
            Don't be afraid to contact me! I'm happy to answer any questions,
            provide more information, or just have a nice conversation! Fill out
            the form below to being reaching out to me. If you prefer another
            email client other than your default, which will appear after
            clicking the button and have all of the information you just
            entered, you can email me at XXXXX.
          </p>
          <div className="about__main">
            <h3>Contact Information</h3>
            <em>And for those that don't fancy reading:</em>
            <p className="stat__details">
              <small>XXXX</small>
              <br></br>
              <small>XXXX</small>
              <br></br>
              <small>(XXXX)-445-7747</small>
              <br></br>
              <small>XXXXXX@gmail.com</small>
            </p>
          </div>
        </div>
        <div className="about__main">
          <h3>right</h3>
          <form onClick={(e) => handelClick(e)}>
            <div className="row">
              <div className="col-sm-6 col-md-6">
                <div className="firstname">
                  <h4>Full Name *</h4>
                  <input type="text" placeholder="First Name"></input>
                </div>
              </div>

              <div className="col-sm-6 col-md-6">
                <div className="lastname">
                  <h4>Full Name *</h4>
                  <input type="text" placeholder="Last Name"></input>
                </div>
              </div>
            </div>

            <div className="row">
              <div className="col-sm-12 col-md-12">
                <div className="email">
                  <h4>Email *</h4>
                  <input type="text" placeholder="Email"></input>
                </div>
              </div>
            </div>

            <div className="row">
              <div className="col-sm-12 col-md-12">
                <div className="message">
                  <h4>Message *</h4>
                  <textarea placeholder="Message"></textarea>
                </div>
              </div>
            </div>

            <div className="row">
              <div className="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4">
                <div className="">
                  <Button scrollTo={0} content={"Submit"} />
                </div>
              </div>
            </div>
          </form>
        </div>
      </main>
    </div>
  );
}

export default Contact;
