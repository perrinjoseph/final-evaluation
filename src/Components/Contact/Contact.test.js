import { render } from "@testing-library/react";
import Contact from "./Contact";
describe("Contact", () => {
  it("should match the snapshot", () => {
    const { asFragment } = render(<Contact>Snapshot Contact</Contact>);
    expect(asFragment()).toMatchSnapshot();
  });
});
