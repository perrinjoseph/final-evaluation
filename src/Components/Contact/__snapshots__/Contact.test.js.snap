// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact should match the snapshot 1`] = `
<DocumentFragment>
  <div
    class="contact "
  >
    <h1
      class="projects__heading__center"
    >
      Contact
      <br />
      <small
        style="font-size: 20px;"
      >
        <i>
          "XXXX"
        </i>
      </small>
      <br />
      <small
        style="font-size: 20px;"
      >
        <i>
          "XXXX"
        </i>
      </small>
    </h1>
    <main
      class="contact__main"
    >
      <div
        class="about__main "
      >
        <h3>
          Reaching out to me
        </h3>
        <p>
          Don't be afraid to contact me! I'm happy to answer any questions, provide more information, or just have a nice conversation! Fill out the form below to being reaching out to me. If you prefer another email client other than your default, which will appear after clicking the button and have all of the information you just entered, you can email me at XXXXX.
        </p>
        <div
          class="about__main"
        >
          <h3>
            Contact Information
          </h3>
          <em>
            And for those that don't fancy reading:
          </em>
          <p
            class="stat__details"
          >
            <small>
              XXXX
            </small>
            <br />
            <small>
              XXXX
            </small>
            <br />
            <small>
              (XXXX)-445-7747
            </small>
            <br />
            <small>
              XXXXXX@gmail.com
            </small>
          </p>
        </div>
      </div>
      <div
        class="about__main"
      >
        <h3>
          right
        </h3>
        <form>
          <div
            class="row"
          >
            <div
              class="col-sm-6 col-md-6"
            >
              <div
                class="firstname"
              >
                <h4>
                  Full Name *
                </h4>
                <input
                  placeholder="First Name"
                  type="text"
                />
              </div>
            </div>
            <div
              class="col-sm-6 col-md-6"
            >
              <div
                class="lastname"
              >
                <h4>
                  Full Name *
                </h4>
                <input
                  placeholder="Last Name"
                  type="text"
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-sm-12 col-md-12"
            >
              <div
                class="email"
              >
                <h4>
                  Email *
                </h4>
                <input
                  placeholder="Email"
                  type="text"
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-sm-12 col-md-12"
            >
              <div
                class="message"
              >
                <h4>
                  Message *
                </h4>
                <textarea
                  placeholder="Message"
                />
              </div>
            </div>
          </div>
          <div
            class="row"
          >
            <div
              class="col-sm-4 col-md-4 col-sm-offset-4 col-md-offset-4"
            >
              <div
                class=""
              >
                <button
                  class="button"
                >
                  Submit
                </button>
              </div>
            </div>
          </div>
        </form>
      </div>
    </main>
  </div>
</DocumentFragment>
`;
